client/lib/media/actions.js

```diff
@@ -51,7 +51,7 @@ MediaActions.fetch = function( siteId, itemId ) {
 	} );
 
 	debug( 'Fetching media for %d using ID %d', siteId, itemId );
-	wpcom.site( siteId ).media( itemId ).get( function( error, data ) {
+	wpcom.site( siteId ).media( itemId ).get( { apiVersion: '1.2' }, function( error, data ) {
```

Comment:
If you switch to a Jetpack site we'll have infinity loaders
![screen shot 2016-10-27 at 10 16 12 am](https://cloud.githubusercontent.com/assets/1270189/19777732/71c2e9ac-9c2e-11e6-92b4-a446ac412a4e.png)